L/LVGL_SDL/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LVGL_SDL"
+version = v"9.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lvgl/lvgl.git", "4f086111a107718b719a45e2cc422f5b756d7edd"),
```

Comment:
Why is this using a git snapshot and not a released version? In particular it seems the latest released version is 9.2.0, so why do you call this snapshot "9.3.0" ?

Reply:
It's true they don't offer stable release tarballs, just the the autogenerated ones (my guess would be that as usual they are not even aware that those archives are not stable). But they still have release tags...

And by the discussion in https://github.com/lvgl/lvgl/issues/7098 and https://github.com/lvgl/lvgl/issues/6763 it seems 9.3.0 is only planned for early 2025, so it seems unlikely to match what is being used here as 9.3.0